lms/product/blackboard/__init__.py

```diff
@@ -1 +1,8 @@
+from lms.product.blackboard._plugin.grouping import BlackboardGroupingPlugin
 from lms.product.blackboard.product import Blackboard
+
+
+def includeme(config):
+    config.register_service_factory(
+        BlackboardGroupingPlugin.factory, iface=BlackboardGroupingPlugin
+    )
```

Comment:
Going for high localism here. We should be able to alter Canvas stuff without editing anything outside this directory ideally.